### PR TITLE
feature: individal query view for analytics

### PR DESCRIPTION
**Pull request by drew-harris**

### CODE REVIEW

**Comment by drew-harris** [comment]
image

**Comment by drew-harris** [comment]
image

**Comment by drew-harris** [comment]
image

**Review by cdxker — APPROVED** [review]

File: frontends/analytics/package.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "tailwind-gradient-mask-image": "^1.2.0",
     "tailwind-scrollbar": "^3.1.0",
     "trieve-fetch-client": "*",
-    "yarn": "^1.22.22"
+    "yarn": "^1.22.22",
+    "zod": "^3.23.8"
```
Comment (cdxker) [inline frontends/analytics/package.json:33]:
```suggestion
```
This doesn't seem to be used, can this be removed?

**Review by cdxker — CHANGES_REQUESTED** [review]

File: frontends/analytics/package.json (hunk 1 above)
Comment (drew-harris) [reply frontends/analytics/package.json:33]: Its used in SIngleQueryInfo to parse the v1 search metadata shape. Probably unnecessary for this use case but its a small dep and will very likely come in handy later based on the direction analytics is going.

**Review by cdxker — APPROVED** [review]

**Pull request merged**